### PR TITLE
[ADD] l10n_jo: Jordan localization

### PR DESCRIPTION
**Pull request by hsal-odoo**
This commit adds a localization module for Jordan. It includes the following:
1. Chart of accounts.
2. Accounts groups.
3. Taxes.
4. Tax report.

task-2732319

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by asfa-odoo** [comment]
Re-target of this PR (https://github.com/odoo/odoo/pull/114138) to 17.0

Thanks @hsal-odoo

File: addons/l10n_jo/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from . import models
+from odoo import api, SUPERUSER_ID
+
+
+def load_translations(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env.ref('l10n_jo.jordan_chart_template_standard').process_coa_translations()
```
Comment (jco-odoo) [inline addons/l10n_jo/__init__.py:7]:
```suggestion
```
No need for that anymore

File: addons/l10n_jo/data/template/account.account-jo_standard.csv [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+"jo_account_500907","Others","500907","expense","False","غير ذلك"
+"jo_account_500908","Other Non-Operating Expenses","500908","expense","False","النفقات الأخرى غير التشغيلية"
+"jo_account_500909","Cash Difference Loss","500909","expense","False","خسائر فريق النقد"
+"jo_account_501101","Telephone","501101","expense","False","الهاتف"
+"jo_account_501102","Others - Communication","501102","expense","False","غير ذلك - التواصل"
+"jo_account_501103","Maintenance","501103","expense","False","الصيانة"
+"jo_account_501104","Security & Guard","501104","expense","False","الأمن والحراسة"
+"jo_account_501105","Cleaning","501105","expense","False","التنظيف"
+"jo_account_501106","Others - Office Various Expenses","501106","expense","False","غير ذلك - نفقات المكتب المختلفة"
+"jo_account_501107","Cash Discount Loss","501107","expense","False",""
```
Comment (jco-odoo) [inline addons/l10n_jo/data/template/account.account-jo_standard.csv:137]: No translation?

File: addons/l10n_jo/demo/demo_company.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>    <data>
```
Comment (jco-odoo) [inline addons/l10n_jo/demo/demo_company.xml:2]:
```suggestion
<odoo>    
    <data>
```

**Review by jco-odoo — COMMENTED** [review]
Please check

**Comment by hsal-odoo** [comment]
@robodoo retry

**Pull request closed without merging**